From 5d4ad9666226362f4147e5a099402df479d52695 Mon Sep 17 00:00:00 2001
From: Algea Cao <algea.cao@rock-chips.com>
Date: Mon, 27 May 2019 10:00:18 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3328: Change inno hdmi phy name

DTS hdmi phy name have to be the same with dt-binding document and
hdmi driver code.

Change-Id: I00edde5b5745832f123b98f41a2fdb52434ce386
Signed-off-by: Algea Cao <algea.cao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328.dtsi b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
index bc3d6cd79db2..d0dbfd3a9ad0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328.dtsi
@@ -995,7 +995,7 @@
 			      "cec",
 			      "hclk_vio";
 		phys = <&hdmiphy>;
-		phy-names = "hdmi_phy";
+		phy-names = "hdmi";
 		pinctrl-names = "default", "gpio";
 		pinctrl-0 = <&hdmi_cec &hdmii2c_xfer &hdmi_hpd>;
 		pinctrl-1 = <&i2c3_gpio>;
-- 
2.35.3

